src/FaustSensors.ts

```diff
@@ -187,7 +187,7 @@ export default class FaustSensors {
                     const x = this.fRange.clip(v);
                     return this.fOffset + x * this.fCoef;
                 }
-                getLowHigh(amin: number, amax: number): InterpolateObject {
+                getLowHigh(): InterpolateObject {
```

Comment:
Why changing the prototype here ?

Reply:
The function is a pure declaration here but arguments are used at other implantation places. Better keep them to make that explicit.

Reply:
In the same FaustSensors.ts file.